web/src/engine/websites/Qiman5.ts

```diff
@@ -0,0 +1,39 @@
+import { Tags } from '../Tags';
+import icon from './Qiman5.webp';
+import { Chapter, DecoratableMangaScraper, Page, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchJSON } from './../platform/FetchProvider';
+
+type APIChapters = {
+    data: {
+        name: string,
+        link : string
+    }[]
+}
+
+@Common.MangaCSS(/^{origin}\/[^.]+\.html$/, 'div.info h1.name_mh')
```

Comment:
The part `[^.]` will also match slashes, is this by intention?

Reply:
Ah, missed this one, you are right